setup.py

```diff
@@ -28,7 +28,7 @@
     ],
     keywords         = ['molecular modeling', 'computational chemistry', 'workflow', 'python interface'],
     python_requires  = '>=3.6',
-    install_requires = ['dill>=0.2.4', 'numpy', 'natsort'],
+    install_requires = ['dill>=0.2.4', 'numpy==1.21', 'scipy==1.8.1', 'natsort'],
```

Comment:
I'd strongly recommend avoiding hard versions pins like these as they can *very* easily choke dependency resolvers when multiple packages are installed.

Reply:
I guess we can try to loosen the restriction here, but I first wanted to see how to get a working version.

Reply:
What happens when you remove the version restriction? One of the issues with pinning here is that you won't find a numpy or scipy version that supports both python 3.6 and 3.10.

Reply:
I guess it's picking up a too new version of scipy for the older builds now, but restricting it further would break the higher builds.